sw/device/silicon_creator/rom_ext/rom_ext_start.S

```diff
@@ -154,9 +154,7 @@ _rom_ext_start_boot:
    *
    * The lowest two bits should be `0b01` to ensure we use vectored interrupts.
    */
-  la   t0, _rom_ext_interrupt_vector
-  andi t0, t0, -4
-  ori  t0, t0, 0b01
```

Comment:
I think you need `ori` instruction , because of that:
`The lowest two bits should be `0b01` to ensure we use vectored interrupts.`

Reply:
Please scratch that, I missed that there is `+1`.